### ISSUE TITLE
Running mask detection using opencv-python==4.4.0.46 on  Django==3.1.2

### ISSUE THREAD

**Issue by SharmaHimanshu1991** [opencv/opencv#19272]
##### System information (version)
 gcc --version
-->

- OpenCV => :grey_question:
- Operating System / Platform => :grey_question:
- Compiler => :grey_question:

##### Detailed description

I have to implement live mask detection using more than one IP camera on Django framework. You can find mask detection code from (https://github.com/Shaligram1234/Python) from GitHub repo. 

Implementing this mask detection code (https://github.com/Shaligram1234/Python) from GitHub in my ubuntu computer. 
I am getting error 
**[h264 @ 0x7f2ca4060940] cabac decode of qscale diff failed at 36 17
[h264 @ `0x7f2ca4060940]` error while decoding MB 36 17, bytestream -6**
 and
ret, frame = cap.read()
.read  function returning false after some  time and continually give false after that. 

**Below are my requriment txt file**
absl-py==0.11.0
appdirs==1.4.3
asgiref==3.2.10
astunparse==1.6.3
CacheControl==0.12.6
cachetools==4.2.0
certifi==2019.11.28
chardet==3.0.4
click==7.1.2
cmake==3.18.4.post1
colorama==0.4.3
contextlib2==0.6.0
cycler==0.10.0
distlib==0.3.0
distro==1.4.0
Django==3.1.2
django-admin-sortable2==0.7.7
django-evostream==0.1b2
dlib==19.21.1
face-recognition==1.3.0
face-recognition-models==0.3.0
flatbuffers==1.12
gast==0.3.3
google-auth==1.24.0
google-auth-oauthlib==0.4.2
google-pasta==0.2.0
grpcio==1.32.0
h5py==2.10.0
html5lib==1.0.1
idna==2.8
imutils==0.5.3
ipaddr==2.2.0
iso8601==0.1.13
joblib==1.0.0
Keras-Preprocessing==1.1.2
kiwisolver==1.3.1
lockfile==0.12.2
m3u8==0.8.0
m3u8-dl==0.2.3
Markdown==3.3.3
matplotlib==3.2.1
msgpack==0.6.2
numpy==1.19.4
oauthlib==3.1.0
opencv-contrib-python==4.4.0.46
opencv-python==4.4.0.46
opt-einsum==3.3.0
packaging==20.3
pep517==0.8.2
Pillow==8.0.1
progress==1.5
protobuf==3.14.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pyparsing==2.4.6
python-dateutil==2.8.1
pytoml==0.1.21
pytz==2020.5
requests==2.22.0
requests-oauthlib==1.3.0
retrying==1.3.3
rsa==4.6
scikit-learn==0.23.1
scipy==1.5.4
six==1.15.0
sk-video==1.1.10
sqlparse==0.4.1
tensorboard==2.4.0
tensorboard-plugin-wit==1.7.0
tensorflow==2.4.0
tensorflow-estimator==2.4.0
termcolor==1.1.0
threadpoolctl==2.1.0
typing-extensions==3.7.4.3
urllib3==1.25.8
webencodings==0.5.1
Werkzeug==1.0.1
wrapt==1.12.1

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [x] I checked the problem with documentation, FAQ, open issues,
       answers.opencv.org, Stack Overflow, etc and have not found solution
   
 - [x] I updated to latest OpenCV version and the issue is still there
   
 - [x] There is reproducer code and related data files: videos, images, onnx, etc

**Comment by ZurMaD** [comment]
for django use 
```
opencv-contrib-python-headless==4.4.0.46
```

instead of those:
```
opencv-contrib-python==4.4.0.46
opencv-python==4.4.0.46
```
